topics.py

```diff
@@ -38,6 +38,7 @@ class Camera(Topic):
     set_overlay = 'camera/set_overlay'
     push_overlays = 'camera/push_overlays'
     errors = '/v3/camera/errors'
+    online_root = '/v3/camera/online'
```

Comment:
Can you make a PR into the `common` library to add this as well?

Reply:
Done
[#4](https://github.com/monash-human-power/common/pull/4)